src/components/explorer/ExplorerSearchResultsTable.js

```diff
@@ -29,11 +29,25 @@ const ExplorerSearchResultsTable = ({
 }) => {
     const { dataset } = useParams();
     const [currentPage, setCurrentPage] = useState(initialCurrentPage || 1);
-    const [numResults] = useState(data.length);
+    const [activeFilters, setActiveFilters] = useState([]);
 
     const [summaryModalVisible, setSummaryModalVisible] = useState(false);
     const [tracksModalVisible, setTracksModalVisible] = useState(false);
 
+    const filteredData = useMemo(() => {
+        return data.filter((item) => {
+            return Object.keys(activeFilters).every((filterKey) => {
+                const filterValues = activeFilters[filterKey] || [];
```

Comment:
object.keys + every + a prop access is a confusing way to write this, when it could just be `Object.entries(activeFilters).every(([filterKey, filterValues]) => ...)`. Also, I'd prefer expression-based functions over the multiple-nested-closure with return style seen here.